packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -276,8 +330,14 @@ async function createReferencePriceFeedForEmp(logger, web3, networker, getTime,
     defaultConfig
   });
 
-  // Infer lookback from liquidation liveness.
-  if (emp && defaultConfig) {
+  // Infer lookback from liquidation liveness if default config does not specify a lookback.
+  // - Explanation: Always setting the `lookback = emp.liquidationLiveness` can potentially introduce problems
+  // if the liquidation liveness is particularly long and the pricefeed to be constructed is a DEX.
+  // - For example: the Zelda Cash contract has a 172800 liveness period (48 hours) and it uses the
+  // BasketSpreadPriceFeed which depends on the BalancerPriceFeed. Updating a DEX pricefeed requires
+  // a web3 call for every block included in the lookback window. Therefore, we override some of the default config
+  // lookback values so that they do not make an impractically large amount of web3 requests.
+  if (emp && defaultConfig && !defaultConfig.lookback) {
```

Comment:
I added this condition so that we can override the default setting of `lookback = liquidationLiveness` for certain EMP's like the Zelda cash ones which have very large liquidationLiveness periods of 172800. This large lookback becomes a problem since one of the pricefeeds to update is the `BalancerPriceFeed` which makes a `web3.eth.getBlock` call for every block within the lookback window. Empirically, this throws web3 "too many requests"-esque errors until I override the lookback to 7200.

I think giving the bots 2 hours of lookback is sufficient since I suspect it unlikely that a liquidation or dispute would be missed for > 2 hours, and we can always manually remove the override if the bots are offline for a large amount of time.